app/javascript/packs/document-capture.tsx

```diff
@@ -138,7 +139,13 @@ const App = composeComponents(
       maxSubmissionAttemptsBeforeNativeCamera: Number(maxSubmissionAttemptsBeforeNativeCamera),
     },
   ],
-  [DocumentCapture, { onStepChange: extendSession }],
+  [
+    DocumentCapture,
+    {
+      onStepChange: extendSession,
+      inPersonFullAddressEntryEnabled: inPersonFullAddressEntryEnabled === 'true',
```

Comment:
I suppose it's fine if it's just used in the top-level component and not passed down with props drilling, but this feels like something we'd want to have in the `InPersonContext.Provider` declared above in the file.

Reply:
Would we then pass the InPersonContext down into the DocumentCapture component? Since that's where the logic for which page is shown happens

Reply:
If we add it to context, then the `DocumentCapture` component can reference it as we do with other IPP context.

https://github.com/18F/identity-idp/blob/0cb42b9dfa19ea63c5403d940d3e24b07f40cd49/app/javascript/packages/document-capture/components/document-capture-troubleshooting-options.tsx#L46

Reply:
I'll go ahead and merge as-is but I'm open to suggestions if this doesn't seem like the right way. There will be follow-up PRs where I can adjust it. Let me know if you want to get on a call and discuss

Reply:
Woops I didn't see your response in between my two comments, I had to refresh the page. Sorry! That looks great -- I'll adjust it in the next PR